Add compactness script and reorganize repo

jonmast opened the pull request:
Add compactness script in it's own folder and move ruby scraper to a separate folder.

jonmast committed:
Add compactness calculation script

This script is copied from
https://github.com/tcfuji/anti-gerry-compactness, thanks to @tcfuji for
this.

jonmast committed:
Reorganize files and add readme

Move ruby scraper into its own directory and add basic README.

tcfuji commented:
Sorry but due to some of the dependencies (pandas), I can't output a clean geojson with a compactness column. For some reason, pandas can only create json strings: http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_json.html

The best I can do for now is just output a list of the compactness measures in a separate file. I'll work on it later today.

tcfuji commented:
Just updated script: https://github.com/tcfuji/anti-gerry-compactness/blob/master/compactness.py

jonmast committed:
Prevent double encoding of geojson

Geopandas 'to_json' already encodes it as a JSON string, so it breaks if
we write the file with json.dump.
